inst/components/accordion.scss

```diff
@@ -0,0 +1,14 @@
+$accordion-box-shadow-active: none !default;
```

Comment:
Don't declarations like this belong in the `defaults` part of a `sass_layer`?

Reply:
Good point, thanks!